ci/jobs/scripts/ssh-setup.sh

```diff
@@ -1,5 +1,2 @@
-#!/usr/bin/env bash
-set -euo pipefail
```

Comment:
In jobs, this script is included in shell jobs with other scripts, resulting in unexpected behavior when the other scripts run. At the very least, it should be run in isolation.